### PR TITLE
[chore] Fix "All" parameter sets

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Add missing "sort" parameter for batch list parameter set (enum)
- Mark unused parameters for various list endpoints as "obsolete"
- Handle de/serializing enums in parameter sets

# Testing

- All unit tests pass
- Batch "all" test updated to include "sort" parameter to confirm enum de/serialization works as expected
  - Re-record cassette due to parameters changing

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: EasyPost/Parameters/Address/All.cs
Comment (jchen293) [inline EasyPost/Parameters/Address/All.cs:62]: Looking at the API docs, it seems like the `BeforeId` and `AfterId` are invalid for retrieving a list of addresses endpoint?

Comment (nwithan8) [reply EasyPost/Parameters/Address/All.cs:62]: Per internal conversations, we've discovered our API docs are outdated on this information.

**Review by jchen293 — APPROVED** [review]
Seems fine to me, please have another review from @Justintime50 before merging to master

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**
